Sites/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <title>GD2 PR2</title>
+    <link rel="stylesheet" href="Styles.css">
+</head>
+
+<body>
+    <div>
+        <div class="container clearfix">
+            <h1>
+                <div class="underline">A Day in Chicago</div>
```

Comment:
1. I'm originally from Chicago, so this was a delight 😁 
1. Instead of adding in an extra `div`, you can just apply the "underline" class to your `h1`:
```html
<h1 class="underline">A Day in Chicago</h1>
```